lib/stanford-mods/version.rb

```diff
@@ -1,6 +1,6 @@
 module Stanford
   module Mods
     # this is the Ruby Gem version
-    VERSION = '2.6.2'.freeze
+    VERSION = '2.6.3'.freeze
```

Comment:
Thought about 2.7, but I _think_ the change here shouldn't be "breaking" and would have expected results for consumers (but happy to go either way)